src/GetMatch.elm

```diff
@@ -27,55 +23,43 @@ matches =
     ]
 
 
-checkMatch : Maybe Int -> Maybe Match
-checkMatch match =
-    andThen (\n -> getAt (n - 1) matches) match
-
-
-unwrapToString : Maybe Int -> String
-unwrapToString maybeInt =
-    unwrap "" String.fromInt maybeInt
-
-
-maybeIntToInt : Maybe Int -> Int
-maybeIntToInt mi =
-    case mi of
-        Nothing ->
-            0
-
-        Just n ->
-            n
-
+type AllianceColor
+    = Blue
+    | Red
 
-getMatch : Maybe Int -> String -> String
-getMatch match station =
-    stationIndex match <| station
 
+type StationNumber
+    = One
+    | Two
+    | Three
 
-stationIndex : Maybe Int -> String -> String
-stationIndex match station =
-    case checkMatch match of
-        Nothing ->
-            "Not a match"
 
-        Just matchData ->
-            if station == "Blue 1" then
-                String.fromInt matchData.blue.one
+type alias AllianceStation =
+    ( AllianceColor, StationNumber )
 
-            else if station == "Blue 2" then
-                String.fromInt matchData.blue.two
 
-            else if station == "Blue 3" then
-                String.fromInt matchData.blue.three
+getTeam : AllianceStation -> Match -> Int
+getTeam chosenStation =
+    let
+        alliance : Match -> Alliance
+        alliance =
+            case chosenStation of
+                ( Red, _ ) ->
+                    .red
```

Comment:
Where the Match type came from?
I don't see it set anywhere
I think you meant to delete it